src/libstd/net/udp.rs

```diff
@@ -579,6 +603,34 @@ impl UdpSocket {
         self.0.recv(buf)
     }
 
+    /// Receives data on the socket from the remote adress to which it is
+    /// connected, without removing that data from the queue.
+    ///
+    /// Successive calls return the same data. This is accomplished by passing
+    /// `MSG_PEEK` as a flag to the underlying `recv` system call.
+    ///
+    /// The `connect` method will connect this socket to a remote address. This
```

Comment:
This bit about `connect` should maybe move to a dedicated "Errors" section. I'd also suggest swapping the order of sentences here.